<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QWidget" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="informationLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="allMessageTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendMessageTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
